core/modules/layout/includes/layout.class.inc

```diff
@@ -691,11 +691,28 @@ class Layout {
             $new_positions[$new_region_name][] = $uuid;
           }
           else {
-            unset($this->content[$uuid]);
+            $orphaned_blocks[] = $uuid;
           }
         }
       }
     }
+    if (!empty($orphaned_blocks)) {
+      end($new_positions);
+      $default_region = isset($new_layout_info['default region']) ? $new_layout_info['default region'] : key($new_positions);
+      $reassigned_blocks = array();
+
+      // Assign orphan blocks to the last region.
+      foreach ($orphaned_blocks as $uuid) {
+        $new_positions[$default_region][] = $uuid;
+        $reassigned_blocks[] = $this->content[$uuid]->getAdminTitle();
+      }
+        backdrop_set_message(t('The <em>@reassigned_block</em> @block been reassigned to "@new_region" region because the old region does not exist in the new template. Edit layout to move to more appropriate region.', array(
```

Comment:
Remove extraneous indentation in this section.

Reply:
...actually something like this @herbdool: https://github.com/herbdool/backdrop/pull/6